generator/README.md

```diff
@@ -56,24 +56,17 @@ Or from `generator/`: `go test ./...`
 
 ### Installing `protoc`: the Protobuf Compiler
 
-The unit tests use `protoc` to parse text `.proto` files. You will need this
-installed in your `$PATH` to run the tests.
+The generator uses the protobuf compiler, `protoc`, with the `--retain_options`
+flag. 
 
-You can find numerous guides on how to install the Protobuf Compiler. Here
-we suggest two approaches:
+Ensure that you have `protoc` installed and that its version is >= v23.0.
```

Comment:
I think the `retain_options` thing will change over time.  Maybe something like:

```suggestion
The generator and its unit tests use `protoc`, the Protobuf compiler. Ensure you have `protoc >= v23.0` installed and it is found via your `$PATH`.
```

Reply:
done and also changes on the contrib guide

Reply:
meh, `--retain_options` is why `>= v23.0` is required. It also shows up in the protobuf error, which could have led someone to this doc.

I would have left it in (but obviously I would say that, because I wrote the sentence initially). It's fine though.